scripts/linux/runE2ETest.sh

```diff
@@ -958,10 +973,10 @@ function usage() {
     echo ' -snitchStorageAccount             Azure blob Sstorage account for store logs used in status email for long haul and stress test.'
     echo ' -snitchStorageMasterKey           Master key of snitch storage account for long haul and stress test.'
     echo ' -snitchTestDurationInSecs         Test duration in seconds for long haul and stress test.'
-    echo ' -loadGen1TransportType            Transport type for LoadGen1 for stress test. Default is amqp.'
-    echo ' -loadGen2TransportType            Transport type for LoadGen2 for stress test. Default is amqp.'
-    echo ' -loadGen3TransportType            Transport type for LoadGen3 for stress test. Default is mqtt.'
-    echo ' -loadGen4TransportType            Transport type for LoadGen4 for stress test. Default is mqtt.'
+    echo ' -transportType1                   Transport type for LoadGen1 and TwinTester1 for stress test. Default is amqp.'
+    echo ' -transportType2                   Transport type for LoadGen2 and TwinTester2 for stress test. Default is amqp.'
+    echo ' -transportType3                   Transport type for LoadGen3 and TwinTester3 for stress test. Default is mqtt.'
+    echo ' -transportType4                   Transport type for LoadGen4 and TwinTester4 for stress test. Default is mqtt.'
```

Comment:
I unified transport types for load-gen and TwinTester, as they should mirror eachother.